docs/email/components.html

```diff
@@ -0,0 +1,528 @@
+</td>
+{% endhighlight %}
+        <div class="stacks-preview--example">
+            <a href="#" style="display: inline-block; border: 5px solid #E1ECF4; margin: 0 1px 4px 0; background: #E1ECF4; color: #39739D; font-family: arial, sans-serif; font-size: 12px; line-height: 12px; text-decoration: none; white-space: nowrap;" class="post-tag">tag-name</a>
+            <a href="#" style="display: inline-block; border: 5px solid #E1ECF4; margin: 0 1px 4px 0; background: #E1ECF4; color: #39739D; font-family: arial, sans-serif; font-size: 12px; line-height: 12px; text-decoration: none; white-space: nowrap;" class="post-tag">tag</a>
+            <a href="#" style="display: inline-block; border: 5px solid #E1ECF4; margin: 0 1px 4px 0; background: #E1ECF4; color: #39739D; font-family: arial, sans-serif; font-size: 12px; line-height: 12px; text-decoration: none; white-space: nowrap;" class="post-tag">another-tag-name</a>
+        </div>
+        <div class="-preview">
+            <table cellspacing="0" class="s-table">
+                <thead>
+                    <tr>
+                        <th scope="col">Type</th>
+                        <th scope="col" class="s-table--cell4">Class</th>
+                        <th scope="col" class="">Default State</th>
+                        <th scope="col" class="">Hover State</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <th scope="row" class="va-middle">Regular</th>
+                        <td class="va-middle">
+                            <div class="grid gs4 fw-wrap">
+                                <code class="grid--cell stacks-code">.post-tag</code>
+                            </div>
+                        </td>
+                        <td class="va-middle px4">
+                            <a href="#" style="display: inline-block; border: 5px solid #E1ECF4; margin: 0 1px 4px 0; background: #E1ECF4; color: #39739D; font-family: arial, sans-serif; font-size: 12px; line-height: 12px; text-decoration: none; white-space: nowrap;" class="post-tag">tag-name</a>
+                        </td>
+                        <td class="va-middle px4">
+                            <a href="#" style="display: inline-block; border: 5px solid #cee0ed; margin: 0 1px 4px 0; background: #cee0ed; color: #39739D; font-family: arial, sans-serif; font-size: 12px; line-height: 12px; text-decoration: none; white-space: nowrap;" class="post-tag">tag-name</a>
+                        </td>
+                    </tr>
+                </tbody>
+            </table>
+        </div>
+        <aside class="s-notice s-notice__info s-notice__outlined mt24" role="status" aria-hidden="false">
```

Comment:
Move notice outside of `div.stacks-preview--example`. Getting a connected box-shadow effect.

![image](https://user-images.githubusercontent.com/1544281/43609796-f2351f7a-9672-11e8-8979-87adfa1dbbab.png)